test/integration/features/proxy.feature

```diff
@@ -0,0 +1,87 @@
+@proxy @minishift-only
+Feature: Minishift can run behind proxy
+  As a user I can use Minishift behind a proxy.
+
+  Scenario: Start behind the proxy
+   Given user starts proxy server and sets MINISHIFT_HTTP_PROXY variable
```

Comment:
Wanted to describe the behaviour and processes which are being done there - the step starts proxy and sets the env variable. Felt like it will just complicate things if these were in 2 separate steps.